docs/kyma/docs/032-gs-cluster-installation.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Cluster Kyma installation
+type: Getting Started
+---
+
+## Overview
+
+This Getting Started guide instructs developers to quickly deploy Kyma on a cluster. Kyma installs on a cluster using a proprietary installer based on a Kubernetes operator.
+
+The document provides the prerequisites, the instructions on how to install Kyma and verify the deployment, and the troubleshooting tips.
+
+## Prerequisites
+
+**NOTE:** The public IPs and DNS records for Istio Ingress and remote environments gateway must exist before Kyma installation. Also, Kyma installer does not support clusters on AWS because AWS does not support assigning static IPs during creating ELB.
```

Comment:
The public IPs and DNS records for Istio Ingress and the remote environment gateway must exist prior to the Kyma installation. Also, the Kyma installer does not support clusters on AWS because AWS does not support static IP assignment during ELB creation.